readme.txt
=== Rhea ===
Contributors:		codeinwp
Tags:				black, gray, red, white, one-column, two-columns, right-sidebar, responsive-layout, custom-background, custom-menu, editor-style, featured-images, threaded-comments, translation-ready
Requires at least:	3.3.0
Tested up to:		4.4

Rhea
== Description ==
Rhea is a modern responsive Wordpress Theme. It's perfect for web agencies, digital studios, corporate, product showcase, personal and business portfolio.

= License =
Rhea WordPress theme, Copyright (C) 2014 ThemeIsle.com
Rhea WordPress theme is licensed under the GPL3.

Unless otherwise specified, all the theme files, scripts and images are licensed under GNU General Public License.
The exceptions to this license are as follows:

* jquery.sticky.js
    Copyright 2014-2016 Anthony Garand http://garand.me
    Dual licensed under MIT and GPL.

* Images	 
	 
	License: All unsplash.com images are licensed under the terms of the Creative Commons Zero, http://creativecommons.org/publicdomain/zero/1.0/ 	  
	   
     * background-rhea.jpg
      Source: https://unsplash.com/photos/-icmOdYWXuQ

* Font Awesome
 
	License: SIL OFL 1.1
	
	URL: http://scripts.sil.org/OFL


CHANGELOG.md

### 1.0.1 - 10/11/2016
**Changes:** 
- First version



inc/hooks.php
<?php

// Testimonials hooks
remove_action( 'zerif_testimonials_header_title', 'zerif_testimonials_header_title_function' );
add_action( 'zerif_testimonials_header_title', 'rhea_testimonials_header_title_function' );

function rhea_testimonials_header_title_function() {

	$zerif_testimonials_title = get_theme_mod('zerif_testimonials_title',__('Testimonials','rhea'));

	if( !empty($zerif_testimonials_title) ):

		echo '<h2>'.wp_kses_post( $zerif_testimonials_title ).'</h2>';

	elseif ( is_customize_preview() ):

		echo '<h2 class="zerif_hidden_if_not_customizer"></h2>';

	endif;

}


remove_action( 'zerif_testimonials_header_subtitle', 'zerif_testimonials_header_subtitle_function' );
add_action( 'zerif_testimonials_header_subtitle', 'rhea_testimonials_header_subtitle_function' );

function rhea_testimonials_header_subtitle_function() {

	$zerif_testimonials_subtitle = get_theme_mod('zerif_testimonials_subtitle');

	if( !empty($zerif_testimonials_subtitle) ):

		echo '<h6 class="section-legend">'.wp_kses_post( $zerif_testimonials_subtitle ).'</h6>';

	elseif ( is_customize_preview() ):

		echo '<h6 class="section-legend zerif_hidden_if_not_customizer"></h6>';

	endif;

}

// PRO
$currentTheme = wp_get_theme();
$currentThemeName = $currentTheme->parent();
if ( $currentThemeName == 'Zerif PRO' ) {
	add_filter( 'zerif_portofolio_box_underline_color_filter', 'rhea_primary_color' );
	add_filter( 'zerif_aboutus_background_filter', 'rhea_white_color' );
	add_filter( 'zerif_testimonials_background_filter', 'rhea_white_color' );
	add_filter( 'zerif_footer_widgets_title_filter', 'rhea_text_color' );
	add_filter( 'zerif_testimonials_header_filter', 'rhea_text_color' );
	add_filter( 'zerif_links_color_hover_filter', 'rhea_primary_color' );
	add_filter( 'zerif_footer_widgets_title_border_bottom_filter', 'rhea_primary_color' );
	add_filter( 'zerif_ourteam_background_filter', 'rhea_blue_color' );
	add_filter( 'zerif_testimonials_box_color_filter', 'rhea_blue_color' );
	add_filter( 'zerif_navbar_color_filter', 'rhea_darker_color' );
	add_filter( 'zerif_ribbon_button_background_filter', 'rhea_primary_color' );
	add_filter( 'zerif_ribbon_button_background_hover_filter', 'rhea_white_color' );
	add_filter( 'zerif_ribbon_button_button_color_filter', 'rhea_white_color' );
	add_filter( 'zerif_ribbon_button_button_color_hover_filter', 'rhea_primary_color' );
	add_filter( 'zerif_ribbon_background_filter', 'rhea_blue_color' );
	add_filter( 'zerif_ribbonright_background_filter', 'rhea_primary_color' );


	add_filter( 'zerif_contacus_background_filters', 'rhea_white_color' );
	add_filter( 'zerif_contacus_header_filter', 'rhea_text_color' );
	add_filter( 'zerif_contacus_button_background_filter', 'rhea_primary_color' );
	add_filter( 'zerif_contacus_button_background_hover_filter', 'rhea_blue_color' );
	add_filter( 'zerif_contacus_button_color_hover_filter', 'rhea_primary_color' );

	add_filter( 'zerif_ribbonright_button_background_filter', 'rhea_transparent_color' );
	add_filter( 'zerif_ribbonright_button_background_hover_filter', 'rhea_white_color' );
	add_filter( 'zerif_ribbonright_button_button_color_hover_filter', 'rhea_primary_color' );

	// Package Section 
	add_filter( 'zerif_packages_background_filter', 'rhea_blue_color' );
	add_filter( 'zerif_packages_header_filter', 'rhea_text_color' );
	add_filter( 'zerif_package_price_background_color_filter', 'rhea_darker_color' );

	// Subscribe Section 
	add_filter( 'zerif_subscribe_background_filter', 'rhea_primary_color' );



	add_filter( 'zerif_footer_socials_hover_filter', 'rhea_primary_color' );
	add_filter( 'zerif_ourteam_socials_hover_filter', 'rhea_primary_color' );
	add_filter( 'zerif_titles_bottomborder_color_filter', 'rhea_primary_color' );

	function rhea_primary_color() {
		return '#2bb6b6';
	}
	function rhea_white_color() {
		return '#fff';
	}
	function rhea_text_color() {
		return '#404040';
	}
	function rhea_darker_color() {
		return 'rgb(35, 40, 45)';
	}
	function rhea_blue_color() {
		return '#f4f7f9';
	}
	function rhea_transparent_color() {
		return 'rgba( 255,255,255,0 )';
	}
}

/*
 * Remove footer address, phone and email sections by making the default values empty
 */
add_filter( 'zerif_address_default_filter','rhea_empty_footer_sections' );
add_filter( 'zerif_address_icon_default_filter','rhea_empty_footer_sections' );
add_filter( 'zerif_email_default_filter','rhea_empty_footer_sections' );
add_filter( 'zerif_email_icon_default_filter','rhea_empty_footer_sections' );
add_filter( 'zerif_phone_default_filter','rhea_empty_footer_sections' );
add_filter( 'zerif_phone_icon_default_filter','rhea_empty_footer_sections' );

function rhea_empty_footer_sections() {
	return '';
}

/* Add custom id to the header tag */
add_filter( 'zerif_header_id_filter','zerif_header_id_filter_function' );
function zerif_header_id_filter_function() {
	return 'sticky-header';
}

/* Add custom class to the header tag */
add_filter( 'zerif_header_class_filter','zerif_header_class_filter_function' );
function zerif_header_class_filter_function() {
	return 'holder-header';
}

/* Close the header tag sooner */
add_action( 'zerif_bottom_header','zerif_bottom_header_function' );
function zerif_bottom_header_function() {
	echo '</header>';
}

/* Display the homepage top menu */
add_action( 'zerif_before_header','zerif_before_header_function' );
function zerif_before_header_function() {
	if ( has_nav_menu('homepage-top') ) {
		echo '<div class="full-navigation">';
		echo '<nav>';
		wp_nav_menu( array('theme_location' => 'homepage-top', 'container' => false, 'menu_class' => 'main-nav-list' ) );
		echo '</nav>';
		echo '</div>';
	}
}

/* Add a second button in Ribbon with bottom button */
add_action( 'zerif_ribbon_with_bottom_button_after_button','zerif_ribbon_with_bottom_button_after_button_function' );
function zerif_ribbon_with_bottom_button_after_button_function() {
	$rhea_second_button_label = get_theme_mod('rhea_second_button_label');
	$rhea_second_button_link = get_theme_mod('rhea_second_button_link');
	if ( $rhea_second_button_label && $rhea_second_button_link ) {
		echo '<div data-scrollreveal="enter right after 0s over 1s" class="zerif-buttons-container">';
			echo '<a href="'.esc_url( $rhea_second_button_link ).'" class="btn btn-primary custom-button outline-btn right-button">'.wp_kses_post( $rhea_second_button_label ).'</a>';
		echo '</div>';
	}elseif ( is_customize_preview() ) {
		echo '<div data-scrollreveal="enter right after 0s over 1s" class="zerif-buttons-container zerif_hidden_if_not_customizer">';
			echo '<a href="" class="btn btn-primary custom-button outline-btn right-button"></a>';
		echo '</div>';
	}
}

sections/about_us.php
<?php zerif_before_about_us_trigger(); ?>

<section class="about-us" id="aboutus">

	<?php zerif_top_about_us_trigger(); ?>

	<div class="container">

	<?php
		// Left Side
		$rhea_left_side_title = get_theme_mod('rhea_about_us_left_title',__('We do','rhea'));
		$rhea_left_side_title_highlighted = get_theme_mod('rhea_about_us_left_title_highlight',__('big things','rhea'));
		$rhea_left_side_description = get_theme_mod('rhea_about_us_left_side_description',__('We love to work with startups because they are as passionate as we are about their products. Whether you need a full website redesign or just some sprucing up of current products, we want to help and make your startup dream a reality.','rhea'));

		$rhea_clients_title = get_theme_mod('rhea_about_us_clients_title', __('OUR HAPPY CLIENTS','rhea'));
		$rhea_clients_description = get_theme_mod('rhea_about_us_clients_description', __('It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal.','rhea'));
	?>

	<div class="row">
		<div class="col-lg-6 col-md-6 col-sm-12 col-xs-12 left-about-us">
			<h2 class="about-us-left-title">
				<?php if ( $rhea_left_side_title != '' ) {
					echo '<span class="normal">'.$rhea_left_side_title.'</span> ';
				}else{ ?>
					<span class="normal zerif_hidden_if_not_customizer"></span>
				<?php } ?>
				<?php if ( $rhea_left_side_title_highlighted != '' ) { ?>
					<span class="colored">
						<?php echo $rhea_left_side_title_highlighted ?>
					</span>
				<?php }else{ ?>
					<span class="colored zerif_hidden_if_not_customizer"></span>
				<?php } ?>
			</h2>
			<?php if ( $rhea_left_side_description != '' ) { ?>
				<p class="about-us-left-description"><?php echo $rhea_left_side_description ?></p>
			<?php }else{ ?>
				<p class="about-us-left-description zerif_hidden_if_not_customizer"></p>
			<?php } ?>
			<div class="progressbar-container">
				<?php if ( is_active_sidebar( 'sidebar-progress-bar' ) ) {
					dynamic_sidebar( 'sidebar-progress-bar' );
				}else{ ?>
					<div class="progress-holder">
						<h3><?php _e('CREATIVITY', 'rhea'); ?></h3>
						<span class="completion-rate" style="width: 70%">70%</span>
						<div class="progress">
						  <div class="progress-bar progress-bar-striped" role="progressbar" aria-valuenow="40" aria-valuemin="0" aria-valuemax="100" style="width: 70%">
						  </div>
						</div>
					</div>
					
					<div class="progress-holder">
						<h3><?php _e('DEVELOPMENT', 'rhea'); ?></h3>
						<span class="completion-rate" style="width: 87%">87%</span>
						<div class="progress">
						  <div class="progress-bar progress-bar-striped" role="progressbar" aria-valuenow="40" aria-valuemin="0" aria-valuemax="100" style="width: 87%">
						    <span class="sr-only">87% Complete (success)</span>
						  </div>
						</div>
					</div>

					<div class="progress-holder">
						<h3><?php _e('MARKETING', 'rhea'); ?></h3>
						<span class="completion-rate" style="width: 93%">93%</span>
						<div class="progress">
						  <div class="progress-bar progress-bar-striped" role="progressbar" aria-valuenow="40" aria-valuemin="0" aria-valuemax="100" style="width: 93%">
						    <span class="sr-only">93% Complete (success)</span>
						  </div>
						</div>
					</div>

					<div class="progress-holder">
						<h3><?php _e('WORDPRESS', 'rhea'); ?></h3>
						<span class="completion-rate" style="width: 95%">95%</span>
						<div class="progress">
						  <div class="progress-bar progress-bar-striped" role="progressbar" aria-valuenow="40" aria-valuemin="0" aria-valuemax="100" style="width: 95%">
						    <span class="sr-only">95% Complete (success)</span>
						  </div>
						</div>
					</div>
				<?php } ?>
			</div>
		</div>

		<div class="col-lg-6 col-md-6 col-sm-12 col-xs-12 aboutus_boxes">
			<?php if ( is_active_sidebar( 'sidebar-right-aboutus' ) ) {
				dynamic_sidebar( 'sidebar-right-aboutus' );
			}else{ ?>
				<div class="about_us_box">
					<div class="header_aboutus_box">
						<div class="pull-left icon-holder">
							<i class="fa fa-lightbulb-o"></i>
						</div>
						<div class="aboutus_titles pull-left">
							<h4><?php _e('Our History', 'rhea'); ?></h4>
							<p><?php _e('Company was founded in 2016', 'rhea'); ?></p>
						</div>
						<div class="clearfix"></div>
					</div>
					<div class="aboutus_content">
						<p><?php _e('It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters.', 'rhea'); ?></p>
					</div>
				</div>
				<div class="about_us_box">
					<div class="header_aboutus_box">
						<div class="pull-left icon-holder">
							<i class="fa fa-lightbulb-o"></i>
						</div>
						<div class="aboutus_titles pull-left">
							<h4><?php _e('Our Mission', 'rhea'); ?></h4>
							<p><?php _e('Company was founded in 2016', 'rhea'); ?></p>
						</div>
						<div class="clearfix"></div>
					</div>
					<div class="aboutus_content">
						<p><?php _e('It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters.', 'rhea'); ?></p>
					</div>
				</div>
			<?php } ?>
		</div>

	</div>

</div> <!-- / END CONTAINER -->

	<?php zerif_bottom_about_us_trigger(); ?>
	
</section> <!-- END ABOUT US SECTION -->

<?php zerif_after_about_us_trigger(); ?>

<section class="clients">
	<div class="clients-container">
		<div class="container">
			<div class="row">
			<!-- CLIENTS -->
			<?php
				if(is_active_sidebar( 'sidebar-aboutus' )):
					echo '<div class="custom-clients col-lg-4 col-md-4 col-sm-6 col-xs-12">';
						if ( $rhea_clients_title != '' ) {
							echo '<h2>'.$rhea_clients_title.'</h2>';
						}else{
							echo '<h2 class="zerif_hidden_if_not_customizer"></h2>';
						}
						
						if ( $rhea_clients_description != '' ) {
							echo '<p>'.$rhea_clients_description.'</p>';
						}else{
							echo '<p class="zerif_hidden_if_not_customizer"></p>';
						}
						
					echo '</div>';

					echo '<div class="custom-client-list col-lg-8 col-md-8 col-sm-6 col-xs-12">';
						echo '<div data-scrollreveal="enter right move 60px after 0.00s over 2.5s">';
						dynamic_sidebar( 'sidebar-aboutus' );
						echo '</div>';
					echo '</div> ';
				endif;
			?>
			</div>
		</div>
	</div>
</section>

functions.php
<?php

function rhea_lite_setup() {   

	remove_theme_support( 'custom-background' );
	$args = array(
		'default-image' => get_stylesheet_directory_uri() . '/images/background-rhea.jpg',
	);
	add_theme_support( 'custom-header', $args );

	// Register new menu
	register_nav_menus(array(
        'homepage-top' => __('Homepage Top menu', 'rhea'),
    ));

    // Zerif Hooks
	require_once 'inc/hooks.php';

	// Hooks
	remove_action( 'zerif_primary_navigation', 'zerif_primary_navigation_function' );
	add_action( 'zerif_primary_navigation', 'rhea_primary_navigation_function' );
	remove_action( 'zerif_big_title_text', 'zerif_big_title_text_function' );
	add_action( 'zerif_big_title_text', 'rhea_big_title_text_function' );

}

add_action('after_setup_theme', 'rhea_lite_setup', 11 );



function rhea_slug_fonts_url() {
    $fonts_url = '';
     /* Translators: If there are characters in your language that are not
    * supported by Lora, translate this to 'off'. Do not translate
    * into your own language.
    */

    $lato = _x( 'on', 'Lato font: on or off', 'rhea' );
    $homemade = _x( 'on', 'Homemade font: on or off', 'rhea' );
    /* Translators: If there are characters in your language that are not
    * supported by Open Sans, translate this to 'off'. Do not translate
    * into your own language.
    */
    $raleway = _x( 'on', 'Raleway font: on or off', 'rhea' );
    $roboto = _x( 'on', 'Roboto font: on or off', 'rhea' );

    $zerif_use_safe_font = get_theme_mod('zerif_use_safe_font');
    
    if ( ( 'off' !== $lato || 'off' !== $playfair || 'off' !== $homemade ) && isset($zerif_use_safe_font) && ($zerif_use_safe_font != 1) ) {
        $font_families = array();

        if ( 'off' !== $lato ) {
            $font_families[] = 'Lato:300,400,700,400italic,700italic';
        }
        if ( 'off' !== $raleway ) {
            $font_families[] = 'Raleway:500,800,100,300,400';
        }
        if ( 'off' !== $roboto ) {
            $font_families[] = 'Roboto:400,100,300,700';
        }
        
        if ( 'off' !== $homemade ) {
            $font_families[] = 'Homemade Apple';
        }
         $query_args = array(
            'family' => urlencode( implode( '|', $font_families ) ),
            // 'subset' => urlencode( 'latin,latin-ext' ),
        );
         $fonts_url = add_query_arg( $query_args, '//fonts.googleapis.com/css' );
    }
     return $fonts_url;
}

add_action( 'wp_enqueue_scripts', 'rhea_dequeue_styles', 11 );
function rhea_dequeue_styles() {
	wp_dequeue_style( 'zerif_font' );
	wp_dequeue_style( 'zerif_fontawesome' );
}

add_action( 'wp_enqueue_scripts', 'rhea_enqueue_styles' );
function rhea_enqueue_styles() {
	wp_enqueue_style('rhea_font', rhea_slug_fonts_url(), array(), null );
	wp_enqueue_style( 'rhea-fontawesome', get_stylesheet_directory_uri() . '/css/font-awesome.min.css' );
	wp_enqueue_style( 'zerif-lite-style', get_template_directory_uri() . '/style.css', array('zerif_bootstrap_style') );

	wp_enqueue_script( 'rhea-sticky-script', get_stylesheet_directory_uri() . '/js/jquery.sticky.js', array('jquery'), '1.0.0', true );
	wp_enqueue_script( 'rhea-scripts', get_stylesheet_directory_uri() . '/js/rhea-scripts.js', array('jquery'), '1.0.0', true );
}

add_action('wp_head','rhea_lite_background');
function rhea_lite_background() {

	$header_image = get_header_image();
	$style = '<style>';
	if ( $header_image ) {
		$style .='body .rhea-header { background-image: url('.$header_image.'); }';

		$rhea_parallax_background = get_theme_mod( 'rhea_parallax_show' );
		if ( $rhea_parallax_background ) {
			$style .="body .rhea-header { background-attachment: fixed; }";
		}
	}

	$currentTheme = wp_get_theme();
	$currentThemeName = $currentTheme->parent();

	if ( $currentThemeName == 'Zerif PRO' ) {

		// [About us] Left Side
		$left_side_background = get_theme_mod( 'rhea_left_side_background_color', '#f4f7f9' );
		$left_side_title = get_theme_mod( 'rhea_left_side_title_color', '#404040' );
		$left_side_hightlight_background = get_theme_mod( 'rhea_left_side_highlight_background_color', '#2bb6b6' );
		$left_side_hightlight = get_theme_mod( 'rhea_left_side_highlight_color', '#FFFFFF' );
		$left_side_progressbar = get_theme_mod( 'rhea_left_side_progressbar_color', '#2bb6b6' );
		$left_side_description = get_theme_mod( 'rhea_left_side_description_color', '#777777' );

		if ( $left_side_background ) {
			$style .="body .about-us:before { background-color: {$left_side_background}; }";
		}
		if ( $left_side_title ) {
			$style .="body .left-about-us h2 span { color: {$left_side_title}; }";
		}
		if ( $left_side_hightlight_background ) {
			$style .="body .left-about-us h2 span.colored { background-color: {$left_side_hightlight_background}; }";
		}
		if ( $left_side_hightlight ) {
			$style .="body .left-about-us h2 span.colored { color: {$left_side_hightlight}; }";
		}
		if ( $left_side_progressbar ) {
			$style .="body .left-about-us .progress-bar { background-color: {$left_side_progressbar}; }";
		}
		if ( $left_side_description ) {
			$style .="body .about-us .left-about-us p { color: {$left_side_description}; }";
		}

		// [About us] Right Side
		$right_side_icon = get_theme_mod( 'rhea_right_side_icon_color', '#777777' );
		$right_side_description = get_theme_mod( 'rhea_right_side_description_color', '#777777' );
		$right_side_title = get_theme_mod( 'rhea_right_side_title_color', '#404040' );

		if ( $right_side_icon ) {
			$style .="body .about_us_box .icon-holder i { color: {$right_side_icon}; }";
		}
		if ( $right_side_description ) {
			$style .="body .about-us p { color: {$right_side_description}; }";
		}
		if ( $right_side_title ) {
			$style .="body .aboutus_titles h4 { color: {$right_side_title}; }";
		}

		// [Client' Section]
		$clients_title = get_theme_mod( 'rhea_clients_title_color', '#404040' );
		$clients_description = get_theme_mod( 'rhea_clients_description_color', '#777777' );

		if ( $clients_title ) {
			$style .="body .custom-clients h2 { color: {$clients_title}; }";
		}
		if ( $clients_description ) {
			$style .="body .custom-clients p { color: {$clients_description}; }";
		}

		$footer_widgets = get_theme_mod( 'rhea_footer_widgets_background', '#f4f7f9' );
		$footer_widgets_border = get_theme_mod( 'zerif_footer_widgets_title_border_bottom', '#2bb6b6' );
		$footer_widgets_title = get_theme_mod( 'zerif_footer_widgets_title', '#404040' );
		if ( $footer_widgets ) {
			$style .="body .footer-widget-wrap { background-color: {$footer_widgets} !important; }";
		}
		if ( $footer_widgets_title ) {
			$style .="body .footer-widget-wrap .widget .widget-title { color: {$footer_widgets_title} !important; }";
		}
		if ( $footer_widgets_border ) {
			$style .="body .widget .widget-title:before { background-color: {$footer_widgets_border} !important; }";
		}

		// General Colors
		$zerif_menu_item_color = get_theme_mod( 'zerif_menu_item_color', '#fff' );
		if( !empty($zerif_menu_item_color) ){
			$style.="body .navbar-inverse .navbar-nav>li>a, body .navbar-inverse .navbar-nav ul.sub-menu li a{
				color:".$zerif_menu_item_color.";
			}";
		}

		// Top CTA Section
		$rhea_top_cta_second_background = get_theme_mod( 'rhea_top_cta_second_button_background', 'rgba( 255,255,255,.0 )' );
		$rhea_top_cta_second_border = get_theme_mod( 'rhea_top_cta_second_button_border', '#2bb6b6' );
		$rhea_top_cta_second_text = get_theme_mod( 'rhea_top_cta_second_button_text', '#2bb6b6' );
		$rhea_top_cta_second_background_hover = get_theme_mod( 'rhea_top_cta_second_button_background_hover', '#2bb6b6' );
		$rhea_top_cta_second_text_hover = get_theme_mod( 'rhea_top_cta_second_button_text_hover', '#FFF' );

		if ( $rhea_top_cta_second_background ) {
			$style .="body .separator-one .outline-btn { background-color: {$rhea_top_cta_second_background} !important; }";
		}
		if ( $rhea_top_cta_second_border ) {
			$style .="body .separator-one .outline-btn { border-color: {$rhea_top_cta_second_border} !important; }";
		}
		if ( $rhea_top_cta_second_background_hover ) {
			$style .="body .separator-one .outline-btn:hover { background-color: {$rhea_top_cta_second_background_hover} !important;border-color: {$rhea_top_cta_second_background_hover} !important; }";
		}
		if ( $rhea_top_cta_second_text ) {
			$style .="body .separator-one .outline-btn { color: {$rhea_top_cta_second_text} !important; }";
		}
		if ( $rhea_top_cta_second_text_hover ) {
			$style .="body .separator-one .outline-btn:hover { color: {$rhea_top_cta_second_text_hover} !important; }";
		}

		// Bottom CTA Section
		$rhea_bottom_cta_border = get_theme_mod( 'rhea_bottom_cta_button_border', '#fff' );
		$rhea_bottom_cta_background = get_theme_mod( 'zerif_ribbonright_button_background_hover', '#fff' );

		if ( $rhea_bottom_cta_border ) {
			$style .="body .purchase-now .outline-btn { border-color: {$rhea_bottom_cta_border} !important; }";
		}
		if ( $rhea_bottom_cta_background ) {
			$style .="body .purchase-now .outline-btn:hover { border-color: {$rhea_bottom_cta_background} !important; }";
		}

	}
	$style .= '</style>';
	echo $style;

}

function rhea_widgets_init() {    

	register_sidebar(array(
        'name' => __('Progress Bar Section', 'rhea'),
        'id' => 'sidebar-progress-bar',
        'description' => 'This sidebar is used on Homepage in About us Section',
        'before_widget' => '<aside id="%1$s" class="widget %2$s">',
        'after_widget' => '</aside>',
        'before_title' => '<h2 class="widget-title">',
        'after_title' => '</h2>',
    ));

    register_sidebar(array(
        'name' => __('Right Section', 'rhea'),
        'id' => 'sidebar-right-aboutus',
        'description' => 'This sidebar is used on Homepage in About us Section',
        'before_widget' => '',
        'after_widget' => '',
        'before_title' => '<h1 class="widget-title">',
        'after_title' => '</h1>',
    ));
    
}

add_action('widgets_init', 'rhea_widgets_init');

require_once 'inc/rhea_customizer.php';

function rhea_customize_preview_js() {
	wp_enqueue_script( 'rhea_customizer', get_stylesheet_directory_uri() . '/js/rhea-customizer.js', array( 'customize-preview' ), '20130508', true );
}
add_action( 'customize_preview_init', 'rhea_customize_preview_js' );


add_filter( 'customizer_widgets_section_args', 'rhea_change_sidebar_customizer_args' );

function rhea_change_sidebar_customizer_args( $section_args ) {

	$currentTheme = wp_get_theme();
	$currentThemeName = $currentTheme->parent();

	if ( $currentThemeName == 'Zerif PRO' ) {
		$panel_about = 'panel_6';
	}else{
		$panel_about = 'panel_about';
	}

	if ( $section_args['sidebar_id'] == 'sidebar-aboutus' ) {
		$section_args['title'] = 'Clients section';
		$section_args['panel'] = $panel_about;
	}elseif ( $section_args['sidebar_id'] == 'sidebar-progress-bar' || $section_args['sidebar_id'] == 'sidebar-right-aboutus' ) {
		$section_args['panel'] = $panel_about;
		$section_args['priority'] = 3;
	}

	return $section_args;

}

// Add extra body class
add_filter('body_class', 'rhea_add_extra_class');

function rhea_add_extra_class( $classes ) {

	$hero_section = get_theme_mod('zerif_bigtitle_show');
	if( is_front_page() && get_option( 'show_on_front' ) != 'page' && $hero_section != 1 ) {
		$classes[] = 'rhea-front-page';
	}

	return $classes;

}

// Widgets
require_once 'inc/widgets/features.widget.php';
require_once 'inc/widgets/about.widget.php';
require_once 'inc/widgets/hours.widget.php';
require_once 'inc/widgets/contact.widget.php';
require_once 'inc/widgets/progress-bar.widget.php';
require_once 'inc/widgets/icon-box.widget.php';
add_action('widgets_init', 'rhea_register_widgets');

function rhea_register_widgets() {
	register_widget('rhea_features_block');
	register_widget('Rhea_Progress_Bar');
	register_widget('Rhea_Icon_Box');
	register_widget('Rhea_About_Company');
	register_widget('Rhea_Hours');
	register_widget('Rhea_Contact_Company');
}

function rhea_load_custom_wp_admin_style() {
		
		wp_enqueue_style( 'fontawesome-style', get_stylesheet_directory_uri() . '/css/font-awesome.min.css' );
		wp_enqueue_style( 'rhea-admin-style', get_stylesheet_directory_uri() . '/css/admin-style.css' );

        wp_enqueue_script( 'fontawesome-icons', get_stylesheet_directory_uri() . '/js/icons.js', false, '1.0.0' );
        wp_enqueue_script( 'jquery-ui-dialog' );
        wp_enqueue_script( 'fontawesome-script', get_stylesheet_directory_uri() . '/js/fontawesome.jquery.js', false, '1.0.0' );
}
add_action( 'admin_enqueue_scripts', 'rhea_load_custom_wp_admin_style' );

add_action('admin_footer', 'rhea_add_html_to_admin_footer');
add_action('customize_controls_print_footer_scripts', 'rhea_add_html_to_admin_footer');
function rhea_add_html_to_admin_footer() { ?>
	<div id="fontawesome-popup">
		<div class="left-side">
			<label for="fontawesome-live-search">Search icon:</label>
			<input type="text" id="fontawesome-live-search">
			<ul class="filter-icons">
				<li data-filter="all" class="active">All Icons</li>
			</ul>
		</div>
		<div class="right-side">
		</div>
	</div>
<?php }

// Hooks functions

function rhea_primary_navigation_function() { ?>
	<nav class="navbar-inverse site-menu">
		<?php wp_nav_menu( array('theme_location' => 'primary', 'container' => false, 'menu_class' => 'main-nav-list navbar-nav pull-right', 'fallback_cb' => 'zerif_wp_page_menu' )); ?>
	</nav>
<?php }

function rhea_big_title_text_function() {

	$zerif_bigtitle_title = get_theme_mod('zerif_bigtitle_title',__('<strong>Rhea</strong> is super awesome','rhea'));
	if( !empty($zerif_bigtitle_title) ):
		echo '<h1 class="intro-text">'.wp_kses_post( $zerif_bigtitle_title ).'</h1>';
	elseif ( isset( $wp_customize ) ):
		echo '<h1 class="intro-text zerif_hidden_if_not_customizer"></h1>';
	endif;	

	$rhea_bigtitle_description = get_theme_mod('rhea_description',__('And is build on <u>Zerif Lite</u> the most popular one page theme from WordPress.org','rhea'));
	if( !empty($rhea_bigtitle_description) ):
		echo '<p class="intro-description">'.wp_kses_post( $rhea_bigtitle_description ).'</p>';
	elseif ( isset( $wp_customize ) ):
		echo '<p class="intro-description zerif_hidden_if_not_customizer"></p>';
	endif;
	
}

/*
 * Remove footer address, phone and email sections by making the default values empty
 */
add_filter( 'zerif_address_default_filter','rhea_empty_footer_sections' );
add_filter( 'zerif_address_icon_default_filter','rhea_empty_footer_sections' );
add_filter( 'zerif_email_default_filter','rhea_empty_footer_sections' );
add_filter( 'zerif_email_icon_default_filter','rhea_empty_footer_sections' );
add_filter( 'zerif_phone_default_filter','rhea_empty_footer_sections' );
add_filter( 'zerif_phone_icon_default_filter','rhea_empty_footer_sections' );

function rhea_empty_footer_sections() {
	return '';
}

/* Add custom id to the header tag */
add_filter( 'zerif_header_id_filter','zerif_header_id_filter_function' );
function zerif_header_id_filter_function() {
	return 'sticky-header';
}

/* Add custom class to the header tag */
add_filter( 'zerif_header_class_filter','zerif_header_class_filter_function' );
function zerif_header_class_filter_function() {
	return 'holder-header';
}

/* Close the header tag sooner */
add_action( 'zerif_bottom_header','zerif_bottom_header_function' );
function zerif_bottom_header_function() {
	echo '</header>';
}

/* Display the homepage top menu */
add_action( 'zerif_before_header','zerif_before_header_function' );
function zerif_before_header_function() {
	if ( has_nav_menu('homepage-top') ) {
		echo '<div class="full-navigation">';
			echo '<nav>';
				wp_nav_menu( array('theme_location' => 'homepage-top', 'container' => false, 'menu_class' => 'main-nav-list' ) );
			echo '</nav>';
		echo '</div>';
	}
}